### PR TITLE
Feature/rtrail 09

### PR DESCRIPTION
**Pull request by DNizhebortsev**
RT-0005 add Entity comment rest layer

### CODE REVIEW

File: src/test/java/io/khasang/rtrail/controller/CommentControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    private final static String ROOT = "http://localhost:8080/comment";
+    private final static String ADD = "/add";
+    private final static String ALL = "/all";
+    private final static String GET_BY_ID = "/get";
+    private final static String UPDATE = "/update";
+    private final static String DELETE = "/delete";
+
+    @Before
+    public void init() {
+
```
Comment (SKuznet) [inline src/test/java/io/khasang/rtrail/controller/CommentControllerIntegrationTest.java]: why empty block is here?

File: src/test/java/io/khasang/rtrail/controller/CommentControllerIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        assertEquals("OK", responseEntity.getStatusCode().getReasonPhrase());
+
+        Comment recComment = responseEntity.getBody();
+        assertNotNull(recComment);
+    }
+
+    @Test
+    public void checkAllComments() {
+        // clean
```
Comment (SKuznet) [inline src/test/java/io/khasang/rtrail/controller/CommentControllerIntegrationTest.java]: ?

**Review by SKuznet — DISMISSED** [review]

File: src/main/java/io/khasang/rtrail/config/AppConfig.java [hunk 3]
```diff
@@ -81,8 +81,8 @@ public RoutDao routDao() {
     }
 
     @Bean
-    public EmployeeDao employeeDao() {
-        return new EmployeeDaoImpl(Employee.class);
+    CommentDao commentDao() {
```
Comment (SKuznet) [inline src/main/java/io/khasang/rtrail/config/AppConfig.java]: why you remove Employee and set @Bean not public?

**Review by SKuznet — DISMISSED** [review]

**Comment by DNizhebortsev** [comment]
I reassembled the branch.

**Review by SKuznet — APPROVED** [review]

**Pull request closed without merging**